cloud/aws/cloudwatchlogs/deleteLogGroup.go

```diff
@@ -0,0 +1,32 @@
+package cloudwatchlogs
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs"
+)
+
+func DeleteLogGroup(ctx context.Context, client *cloudwatchlogs.Client, req DeleteLogGroupRequest) DeleteLogGroupResponse {
+	request := &cloudwatchlogs.DeleteLogGroupInput{
+		LogGroupName: aws.String(req.Name),
+	}
+	response, err := client.DeleteLogGroup(ctx, request)
+	return DeleteLogGroupResponse{
+		response: response,
+		error:    err,
+	}
+}
+
+type DeleteLogGroupRequest struct {
+	Name string
```

Comment:
I would say that It could be better to make attributes private instead of public. We can create a default constructor that retrieve all the attributes and see what we can do it later